firmware/src/networking/mod.rs

```diff
@@ -4,20 +4,21 @@ pub mod wifi;
 mod packets;
 pub use self::packets::Packets;
 
-pub async fn network_task(packets: &Packets) {
+pub async fn network_task(packets: &Packets) -> ! {
 	#[cfg(feature = "net-wifi")]
 	self::wifi::ඞ::network_task(packets).await;
 	#[cfg(feature = "net-stubbed")]
 	stubbed_network_task(packets).await;
+	unreachable!("No network implementation specified")
```

Comment:
Sorry for the typo in my suggested fix! 
I think this fixes it without any need for the `unreachable!()` macro though. Just needed to remove the semicolon.
```suggestion
	stubbed_network_task(packets).await
```